GETTING_STARTED.md

```diff
@@ -0,0 +1,169 @@
+
+**Note:** If there are any perceived or real conflicts between this document
+and the OCI Charter, the OCI Charter takes precedence as the official governing
+document of the Open Container Initiative.
+
+## What is the OCI?
+
+The [OCI website](https://opencontainers.org) has since inception carried a
+mission statement that defined us as "an open governance structure for the
+express purpose of creating open industry standards around container formats
+and runtime." Created in 2015, the core initial purpose was to align Docker and
+other ecosystem players around a runtime and image specification that would
+become an industry baseline for how images are packaged and executed in OCI
+compliant runtimes.
+
+Since that initial work, finalized in 1.0 specs in the summer of 2017, a
+distribution specification is now underway, based on the HTTP API of the initial
+Docker distribution (registry) project. In 2019, an "artifacts" project was
+approved by the TOB and will be used to define other artifact types (in addition
+to OCI images) which may be defined and stored in registries conforming to the
+OCI distribution spec.
+
+## Who uses the OCI?
+
+It makes the most sense to see consumers of OCI as fitting into a few specific
+categories: contributors/members, implementors, and end users.
+
+**Contributors**, including the project maintainers, and member companies have a
+vested interest in bringing forward the "state of the art" with respect to the
+scope of the OCI. Currently this is of course limited to specifications around
+runtime, image, and distribution of containers. The artifacts repository and
+project is related to both image and distribution and is a natural expansion
+of OCI into ongoing support of additional types within registries and related
+cloud native tools. Specifically, [artifacts](https://github.com/opencontainers/artifacts)
+expands the concept around the **what** when discussing non-OCI image content
+(Helm charts, Singularity container images, SPDX, source bundles, etc.) and
+registries, and allows for further experimentation with arbitrary file types
+within the context of this stable andd well-defined capability.
+
+**Implementors** own projects outside the OCI for which they have determined
+value in being "OCI compliant"; whether that's a registry, a unique container
+runtime, or an end-user tool that talks to registries. They take the OCI
+specifications and implement support for them in their project, and potentially
+will use the conformance suite within the OCI to validate their compliance to
+the specification(s).
+
+**End Users** tend to gain value from the OCI specifications in an indirect
+way: they will expect projects and products that claim OCI compliance to
+interoperate smoothly with other projects and products which are OCI compliant.
+They will look to the OCI to continue maturing conformance and specifications
+to best support the cloud native ecosystem's goal of interoperability for
```

Comment:
nit: I'd avoid "cloud native" as I think it's important for us to consider OCI in the context of non-cloud use-cases.  

```suggestion
to best support the container ecosystem's goal of interoperability for
```

Reply:
Not sure cloud native is really tied to the cloud, per se. Yes, literally the word is there. But, is someone using CNCF projects on-prem not "cloud native"? :) This is also the tension we're seeing around OCI being tightly coupled to containers, while the distribution-spec and artifacts enable us to store additional non-container artifacts in a cloud-native/container centric deployment model. 
Anyway, just suggesting this wording cloud native helps our broader cause here.